scripts/create_compdb.py

```diff
@@ -143,12 +150,20 @@ def main() -> None:
         action="store_true",
         help="Dumps the full list of generated files (default: False)",
     )
+    parser.add_argument(
+        "--extra-bazel-flags",
```

Comment:
nit: you specify this each time with one flag
```suggestion
        "--extra-bazel-flag",
```